tests/elements.rs

```diff
@@ -68,7 +67,7 @@ async fn element_tag_name(mut c: Client, port: u16) -> Result<(), error::CmdErro
     let sample_url = sample_page_url(port);
     c.goto(&sample_url).await?;
     let mut elem = c.find(Locator::Id("checkbox-option-1")).await?;
-    assert_eq!(elem.tag_name().await?.to_lowercase(), "input");
+    assert!(elem.tag_name().await?.eq_ignore_ascii_case("input"));
```

Comment:
Bah, just realized that this won't tell us what the tag name _was_ if the test fails. Let's either leave it the way it was, or:
```suggestion
    let tag_name = elem.tag_name().await?;
    assert!(tag_name.eq_ignore_ascii_case("input"), "{} != input", tag_name);
```